x/gamm/keeper/swap_test.go

```diff
@@ -188,11 +192,15 @@ func (suite *KeeperTestSuite) TestBalancerPoolSimpleSwapExactAmountOut() {
 			spotPriceBefore, err := keeper.CalculateSpotPrice(suite.Ctx, poolId, test.param.tokenInDenom, test.param.tokenOut.Denom)
 			suite.NoError(err, "test: %v", test.name)
 
+			prevGasConsumed := suite.Ctx.GasMeter().GasConsumed()
 			tokenInAmount, err := keeper.SwapExactAmountOut(suite.Ctx, suite.TestAccs[0], poolId, test.param.tokenInDenom, test.param.tokenInMaxAmount, test.param.tokenOut)
 			suite.NoError(err, "test: %v", test.name)
 			suite.True(tokenInAmount.Equal(test.param.expectedTokenInAmount),
 				"test: %v\n expect_eq actual: %s, expected: %s",
 				test.name, tokenInAmount, test.param.expectedTokenInAmount)
+			gasConsumedForSwap := suite.Ctx.GasMeter().GasConsumed() - prevGasConsumed
+			// We consume 10,000 gas directly, so the extra I/O operation mean we end up consuming more.
+			suite.Assert().Greater(gasConsumedForSwap, uint64(10000))
```

Comment:
Since we plan on turning the fixed cost into a param/will likely change it at some point, I wonder if it would make sense to reference `GasFeeForSwap` in `x/gamm/types/constants.go` directly here instead of hard-coding 10,000?